install/operator/api/istioinstaller_types.proto

```diff
@@ -0,0 +1,624 @@
+// TODO:
```

Comment:
Reading through these, I see there are two broad categories of things in this file:

- Physical configuration state for individual components (Pilot, Citadel, etc)

- Logical somewhat global behavior configuration (TelemetryConfig, etc)

How about splitting those us in two separate files? Say _components.protos and _control.proto